docs/validators/validator-faq.md

```diff
@@ -199,9 +229,9 @@ If a validator misbehaves, its bonded stake along with its delegators' stake and
 
 * **Double signing:** If someone reports on chain A that a validator signed two blocks at the same height on chain A and chain B, this validator will get slashed on chain A
```

Comment:
I don't think this is right. Double signing is when you sign two blocks at the same height on the same chain. (I.e. blocks 0 through n-1 are equal, two different blocks signed at height n)

Reply:
Yes I recently changed the definition in other docs I'm working on, but forgot to replicate here.

What about "Signing two blocs at the same height that have a common ancestor"? @ValarDragon